### ISSUE TITLE
Do I need to use RI_render_DPR if I am to try testNetwork_demo_1024.py?

### ISSUE THREAD

**Issue by morawi** [zhhoper/DPR#12]
Thanks for sharing this great work!

Do I need to use RI_render_DPR if I want to try testNetwork_demo_1024.py on some 2D images? (Instead of Obama's?)

I gave the code a go and did see any landmark, albedo, normal, uv map and semantic segmentation loaded. Or these landmarks are only needed for training a new model?

Thanks in advance

**Comment by VinayTurpati** [comment]
No, If you want to test images, Just this repo is enough to transform any image.

**Comment by morawi** [comment]
Apparently, this repo only works with face images. So, again, for other types of images (like a whole person in the wild) we need to use  RI_render_DPR.

**Comment by zhhoper** [comment]
You don't need to use RI_render_DPR to use this repo for rendering face images of 1024x1024. However, if you want to work on images other than faces, you may need to train your own network. In that sense, you will need RI_render_DPR. Note that we use spherical harmonics lighting, which assumes the object to be roughly convex and the material of the object to be Lambertian. For person images, you can still try to use these assumptions but they may not satisfy these as assumptions well, for example, the shape of the person may not be convex and the material of the clothes may not be Lambertian. You may want to take a look at this paper if you want to relight a person. https://arxiv.org/pdf/1908.02714.pdf
